src/accordian.js

```diff
@@ -0,0 +1,13 @@
+ export default function() {
+  var btn = $('.accordion__bar');
+  var list = $('.accordion__bar-list');
+
+  btn.on('click', function() {
+    $(this).next(list).slideToggle();
```

Comment:
Try not to use the `next` chain. It may lead to unexpected results.

Instead look at using `siblings`.

This will add some extra steps to find things in the DOM, but will be clearer what your intent is.